src/cachers/index.js

```diff
@@ -0,0 +1,32 @@
+if (process.env.NODE_ENV !== 'production') {
+  require('dotenv').config();
+}
+
+const commands = [
+  'entities:organisations:get', 'entities:organisations:set'
+];
+
+const cli = commands.reduce((memo, command) => {
+  const commandPath = command.replace(/:/g, '/');
+  memo[command] = require(`./${commandPath}`);
+  return memo;
+}, {});
+
+const main = () => {
+  return cli[process.argv[2]].cli();
+};
+
+main()
+  .then(({ body }) => {
+    if (typeof body === 'string') {
+      console.log(`SUCCESS: ${body}`);
+    } else {
+      console.log('SUCCESS:');
+      console.log(JSON.stringify(body, null, 2));
```

Comment:
other than debugging the functionality introduced with this PR, is there actually a purpose to logging the `get` response to the console?

Reply:
No.